document-service/swagger.json

```diff
@@ -0,0 +1,159 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "description" : "Documents API for U.S. Tax Court EF-CMS",
+    "version" : "2018-10-25T19:16:39Z",
+    "title" : "will-ef-cms",
```

Comment:
Likewise to this, but since it's swagger, it seems fine to let it go for now and let @waldoj decide how to handle it when he gets back.  I mean, everything should work fine, except possibly a swagger-ui instance using the `host` field from the next comment.